### PR TITLE
#48 - Footer Refactor

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

File: android/app/src/main/java/com/flexwaysfrs/MainActivity.java [hunk 1]
```diff
@@ -68,7 +68,14 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
   */
   @Override
   public boolean dispatchKeyEvent(KeyEvent event) {
-    if (event.getKeyCode() == KeyEvent.KEYCODE_ENTER) {
+    int KEYCODE_FN = 1017;
```
Comment (mathvaillant) [inline android/app/src/main/java/com/flexwaysfrs/MainActivity.java:71]: Android has no native mapping for this key, that's why.

**Comment by mathvaillant** [comment]
[BUG - Fn key is not usable in the keyboard as it switches the keyboard context](https://trello.com/c/LtNsAE6d/52-fn-key-is-not-usable-in-the-keyboard-as-it-switches-the-keyboard-context)

**Comment by mathvaillant** [comment]
[Footer hot keys refactor](https://trello.com/c/oeC9qv6e/48-footer-hot-keys-refactor)

File: src/screens/Payment.tsx [hunk 2]
```diff
@@ -89,6 +86,7 @@ export function PaymentScreen({ route }: Props) {
       });
     } finally {
       setIsLoading(false);
+      replace(NavigationScreens.MAIN_MENU);
```
Comment (mathvaillant) [inline src/screens/Payment.tsx:89]: Trello ticket n #50

**Review by joaoqueiroz95 — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]
LGTM!

Lets avoid in the future adding all tickets together, it makes it hard to understand if the AC of the card where met

**Pull request merged**